openshift/3.11/pipelines/Jenkinsfile.cicd

```diff
@@ -29,7 +29,7 @@ pipeline {
     API_DIRECTORY = "backend"
     FRONTEND_DIRECTORY = "frontend"
     MAINTENANCE_DIRECTORY = "maintenance"
-    DEVOPS_DIRECTORY = "openshift/3.11"
+    DEVOPS_DIRECTORY = "openshift/4.0"
```

Comment:
You shouldn't need to update this as it is in the `/3.11/` folder.  There should be a new file in the `/4.0/` folder with the update.  We'll clean out 3.11 once everything works.